src/app/components/ATIAnalytics/index.test.tsx

```diff
@@ -1112,7 +1112,7 @@ describe('ATI Analytics Container', () => {
         isAmp: false,
         pageData: articleDataNews,
         pageType: ARTICLE_PAGE,
-        service: 'mundo',
+        service: 'hausa', // this is a temporary fix for this unit test failing as this service will have reverb enabled in the last batch
```

Comment:
Are we amending / removing the code comment in a clean-up phase or can this be removed?

Reply:
Either we will not need this test anymore when all the batches have been released, so we will keep this until the last reverb batch is removed, or if we want to keep the test for the possibility of reverb being disabled on services intentionally, then we will need to change the test to include a forced useReverb: false.

I imagine we won't need the test anymore once we have reverb on all services.

Reply:
I think we should keep the comment to keep things clear for the action in the last batch.